spec/index.bs

```diff
@@ -2296,11 +2305,15 @@ interface GPUAdapter {
                         most or all cases when this occurs. Applications should perform
                         reinitialization logic starting with {{GPU/requestAdapter()}}.
```

Comment:
Just noticed there are several rejections in the text above here which we'll want to update at some point too.

Reply:
Thanks for the reminder. Addressed those as well. I'd appreciate one last review on those prior to merging.